Feature/global ens

BinbinZhou-NOAA opened the pull request:
## Pull Request Testing ##

- [ y] Describe testing already performed for this Pull Request:</br>

- [ y] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ y] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [y ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ y] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ y] Review the source issue metadata (required labels, projects, and milestone).
- [ y] Complete the PR description above.
- [y ] Ensure the PR title matches the feature branch name.
- [y ] Check the following:
- [ y]  Instructions provided on how to run
- [y ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ y]  Check that the ecf file has all the proper definitions of variables
- [y ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ y]  Check to see that the output directory structure is followed
- [ y]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [y ] After submitting the PR, select **Development** issue with the original issue number.
- [y ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [y ] Close the linked issue.

AliciaBentley-NOAA commented:
Thanks for submitting this new PR, @BinbinZhou-NOAA. Can you please send me examples of your final graphics? I'll need to approve them before your graphics Pull Request can be merged. Your final graphics don't need to be on a fancy webpage or anything like that. You can just put your graphics in a directory on rzdm and send me the link to it. Thanks!

BinbinZhou-NOAA commented:
Alicia,

   My primary plots are in rzdm
directory /home/people/emc/www/htdocs/bzhou/evs_plots/gens
In this directory,  further sub-directoies are
/grid2grid  /grid2obs  /grid2obs_init_separate  /headline  /precip
/profile1  /profile2  /sea_ice  /snowfall  /spatial  /sst
There are too many files (over 6000 plots just for 31days)
If you look at them online, you can first
https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens
then type one of above sub-directory/one of png file name
example:
https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.nhem_glb.png


For other plots in /grid2grid,  just change the plot file names

Binbin





On Tue, Jan 3, 2023 at 1:44 PM Alicia Bentley ***@***.***>
wrote:

> Thanks for submitting this new PR, @BinbinZhou-NOAA
> <https://github.com/BinbinZhou-NOAA>. Can you please send me examples of
> your final graphics? I'll need to approve them before your graphics Pull
> Request can be merged. Your final graphics don't need to be on a fancy
> webpage or anything like that. You can just put your graphics in a
> directory on rzdm and send me the link to it. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370106559>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFPJVNHDTT6DYDJRPITWQRXPFANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Binbin- Thanks! Can you please do one more thing for me?

1) cd into your /gens image directory on rzdm
2) vi .htaccess
3) Type Options +Indexes in .htaccess and then save the file (:wq)

Let me know when you've done this. Thanks!
___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Tue, Jan 3, 2023 at 3:23 PM BinbinZhou-NOAA ***@***.***>
wrote:

> Alicia,
>
> My primary plots are in rzdm
> directory /home/people/emc/www/htdocs/bzhou/evs_plots/gens
> In this directory, further sub-directoies are
> /grid2grid /grid2obs /grid2obs_init_separate /headline /precip
> /profile1 /profile2 /sea_ice /snowfall /spatial /sst
> There are too many files (over 6000 plots just for 31days)
> If you look at them online, you can first
> https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens
> then type one of above sub-directory/one of png file name
> example:
>
> https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.nhem_glb.png
>
>
> For other plots in /grid2grid, just change the plot file names
>
> Binbin
>
>
>
>
>
> On Tue, Jan 3, 2023 at 1:44 PM Alicia Bentley ***@***.***>
> wrote:
>
> > Thanks for submitting this new PR, @BinbinZhou-NOAA
> > <https://github.com/BinbinZhou-NOAA>. Can you please send me examples of
> > your final graphics? I'll need to approve them before your graphics Pull
> > Request can be merged. Your final graphics don't need to be on a fancy
> > webpage or anything like that. You can just put your graphics in a
> > directory on rzdm and send me the link to it. Thanks!
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370106559>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFPJVNHDTT6DYDJRPITWQRXPFANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370190673>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBEH42MMSBG3JE63BB3WQSDCFANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

PerryShafran-NOAA commented:
Binbin  - can you describe the changes for this particular PR?

Perry

BinbinZhou-NOAA commented:
Alicia,

   Yes, I did that. See the file
/home/people/emc/www/htdocs/bzhou/evs_plots/gens/.htaccess

Binbin

On Tue, Jan 3, 2023 at 3:29 PM Alicia Bentley ***@***.***>
wrote:

> Binbin- Thanks! Can you please do one more thing for me?
>
> 1) cd into your /gens image directory on rzdm
> 2) vi .htaccess
> 3) Type Options +Indexes in .htaccess and then save the file (:wq)
>
> Let me know when you've done this. Thanks!
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Physical Scientist | Model Evaluation Group
> <https://www.emc.ncep.noaa.gov/users/meg/home/>
> Global Model Verification Team Lead
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Tue, Jan 3, 2023 at 3:23 PM BinbinZhou-NOAA ***@***.***>
> wrote:
>
> > Alicia,
> >
> > My primary plots are in rzdm
> > directory /home/people/emc/www/htdocs/bzhou/evs_plots/gens
> > In this directory, further sub-directoies are
> > /grid2grid /grid2obs /grid2obs_init_separate /headline /precip
> > /profile1 /profile2 /sea_ice /snowfall /spatial /sst
> > There are too many files (over 6000 plots just for 31days)
> > If you look at them online, you can first
> > https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens
> > then type one of above sub-directory/one of png file name
> > example:
> >
> >
> https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.nhem_glb.png
> >
> >
> > For other plots in /grid2grid, just change the plot file names
> >
> > Binbin
> >
> >
> >
> >
> >
> > On Tue, Jan 3, 2023 at 1:44 PM Alicia Bentley ***@***.***>
> > wrote:
> >
> > > Thanks for submitting this new PR, @BinbinZhou-NOAA
> > > <https://github.com/BinbinZhou-NOAA>. Can you please send me examples
> of
> > > your final graphics? I'll need to approve them before your graphics
> Pull
> > > Request can be merged. Your final graphics don't need to be on a fancy
> > > webpage or anything like that. You can just put your graphics in a
> > > directory on rzdm and send me the link to it. Thanks!
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370106559>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFPJVNHDTT6DYDJRPITWQRXPFANCNFSM6AAAAAATP65RZI
> > >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> >
> > Binbin Zhou
> >
> > Lynker at NOAA/NWS/NCEP/EMC
> >
> > 5830 University Research Ct.
> >
> > College Park, MD 20740
> >
> > ***@***.***
> >
> > 301-683-3683
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370190673>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBEH42MMSBG3JE63BB3WQSDCFANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370195506>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFIGLQDMBAAHH4NOJP3WQSDZXANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Perry,

   1) updated GHRSST data in dcom to the new data path
   2) added more domains in 24hr APCP verification conf files since they
are missing

Binbin




On Tue, Jan 3, 2023 at 3:30 PM PerryShafran-NOAA ***@***.***>
wrote:

> Binbin - can you describe the changes for this particular PR?
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370196597>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJVNNZXBA3HEYLJC3TWQSD6XANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Hi Binbin. I checked the contents of your .htaccess document, and it only
contains "+ Indexes". Can you please copy/paste the text below into your
.htaccess document instead? This will allow me to view a list of your final
EVS images. Thanks!

The text to copy/paste into .htaccess is:
Options +Indexes

___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Tue, Jan 3, 2023 at 3:53 PM BinbinZhou-NOAA ***@***.***>
wrote:

> Perry,
>
> 1) updated GHRSST data in dcom to the new data path
> 2) added more domains in 24hr APCP verification conf files since they
> are missing
>
> Binbin
>
>
>
>
> On Tue, Jan 3, 2023 at 3:30 PM PerryShafran-NOAA ***@***.***>
> wrote:
>
> > Binbin - can you describe the changes for this particular PR?
> >
> > Perry
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370196597>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFJVNNZXBA3HEYLJC3TWQSD6XANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370213859>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBBYG7VX5DRGZI3CS6LWQSGVHANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

BinbinZhou-NOAA commented:
Alicia,

  Done. Please double check it.

Binbin

On Thu, Jan 5, 2023 at 2:03 PM Alicia Bentley ***@***.***>
wrote:

> Hi Binbin. I checked the contents of your .htaccess document, and it only
> contains "+ Indexes". Can you please copy/paste the text below into your
> .htaccess document instead? This will allow me to view a list of your final
> EVS images. Thanks!
>
> The text to copy/paste into .htaccess is:
> Options +Indexes
>
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Physical Scientist | Model Evaluation Group
> <https://www.emc.ncep.noaa.gov/users/meg/home/>
> Global Model Verification Team Lead
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Tue, Jan 3, 2023 at 3:53 PM BinbinZhou-NOAA ***@***.***>
> wrote:
>
> > Perry,
> >
> > 1) updated GHRSST data in dcom to the new data path
> > 2) added more domains in 24hr APCP verification conf files since they
> > are missing
> >
> > Binbin
> >
> >
> >
> >
> > On Tue, Jan 3, 2023 at 3:30 PM PerryShafran-NOAA ***@***.***>
> > wrote:
> >
> > > Binbin - can you describe the changes for this particular PR?
> > >
> > > Perry
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370196597>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFJVNNZXBA3HEYLJC3TWQSD6XANCNFSM6AAAAAATP65RZI
> > >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> >
> > Binbin Zhou
> >
> > Lynker at NOAA/NWS/NCEP/EMC
> >
> > 5830 University Research Ct.
> >
> > College Park, MD 20740
> >
> > ***@***.***
> >
> > 301-683-3683
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370213859>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBBYG7VX5DRGZI3CS6LWQSGVHANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1372615528>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFNXCBVNSFNX7TSLL7LWQ4LHNANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Hi Binbin. It works now! Thank you!

-grid2grid/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/>05-Jan-2023
21:14
-grid2obs/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2obs/>05-Jan-2023
21:24
-grid2obs_init_separate/
<https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2obs_init_separate/>19-Dec-2022
21:28
-headline/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/headline/>03-Jan-2023
21:55
-precip/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/precip/>30-Dec-2022
21:34
-profile1/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/profile1/>23-Dec-2022
20:53
-profile2/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/profile2/>25-Dec-2022
01:07
-sea_ice/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/sea_ice/>22-Dec-2022
15:34
-snowfall/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/snowfall/>23-Dec-2022
17:52
-spatial/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/spatial/>05-Jan-2023
21:26
-sst/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/sst/>19-Dec-2022
22:44

___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Thu, Jan 5, 2023 at 4:29 PM BinbinZhou-NOAA ***@***.***>
wrote:

> Alicia,
>
> Done. Please double check it.
>
> Binbin
>
> On Thu, Jan 5, 2023 at 2:03 PM Alicia Bentley ***@***.***>
> wrote:
>
> > Hi Binbin. I checked the contents of your .htaccess document, and it only
> > contains "+ Indexes". Can you please copy/paste the text below into your
> > .htaccess document instead? This will allow me to view a list of your
> final
> > EVS images. Thanks!
> >
> > The text to copy/paste into .htaccess is:
> > Options +Indexes
> >
> > ___________________________________
> > *Alicia M. Bentley, Ph.D.*
> > Physical Scientist | Model Evaluation Group
> > <https://www.emc.ncep.noaa.gov/users/meg/home/>
> > Global Model Verification Team Lead
> > NOAA/NWS/NCEP/EMC (VPPPG Branch)
> > Work Phone: ‪(301) 683-3783‬
> >
> >
> > On Tue, Jan 3, 2023 at 3:53 PM BinbinZhou-NOAA ***@***.***>
> > wrote:
> >
> > > Perry,
> > >
> > > 1) updated GHRSST data in dcom to the new data path
> > > 2) added more domains in 24hr APCP verification conf files since they
> > > are missing
> > >
> > > Binbin
> > >
> > >
> > >
> > >
> > > On Tue, Jan 3, 2023 at 3:30 PM PerryShafran-NOAA ***@***.***>
> > > wrote:
> > >
> > > > Binbin - can you describe the changes for this particular PR?
> > > >
> > > > Perry
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370196597>,
> or
> > > > unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFJVNNZXBA3HEYLJC3TWQSD6XANCNFSM6AAAAAATP65RZI
> > > >
> > > > .
> > > > You are receiving this because you were mentioned.Message ID:
> > > > ***@***.***>
> > > >
> > >
> > >
> > > --
> > >
> > > Binbin Zhou
> > >
> > > Lynker at NOAA/NWS/NCEP/EMC
> > >
> > > 5830 University Research Ct.
> > >
> > > College Park, MD 20740
> > >
> > > ***@***.***
> > >
> > > 301-683-3683
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370213859>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/APUWOBBYG7VX5DRGZI3CS6LWQSGVHANCNFSM6AAAAAATP65RZI
> > >
> > > .
> > > You are receiving this because you commented.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1372615528>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFNXCBVNSFNX7TSLL7LWQ4LHNANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1372799762>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBDO5PJ6GWDERCLVSQTWQ44LNANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

BinbinZhou-NOAA commented:
Alicia,

  Thanks for setting up these display links. Those plots are primary and
only for 31days. The 90 days plots will come soon.





On Thu, Jan 5, 2023 at 4:34 PM Alicia Bentley ***@***.***>
wrote:

> Hi Binbin. It works now! Thank you!
>
> -grid2grid/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/
> >05-Jan-2023
> 21:14
> -grid2obs/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2obs/
> >05-Jan-2023
> 21:24
> -grid2obs_init_separate/
> <
> https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2obs_init_separate/
> >19-Dec-2022
> 21:28
> -headline/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/headline/
> >03-Jan-2023
> 21:55
> -precip/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/precip/
> >30-Dec-2022
> 21:34
> -profile1/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/profile1/
> >23-Dec-2022
> 20:53
> -profile2/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/profile2/
> >25-Dec-2022
> 01:07
> -sea_ice/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/sea_ice/
> >22-Dec-2022
> 15:34
> -snowfall/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/snowfall/
> >23-Dec-2022
> 17:52
> -spatial/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/spatial/
> >05-Jan-2023
> 21:26
> -sst/ <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/sst/>19-Dec-2022
> 22:44
>
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Physical Scientist | Model Evaluation Group
> <https://www.emc.ncep.noaa.gov/users/meg/home/>
> Global Model Verification Team Lead
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Thu, Jan 5, 2023 at 4:29 PM BinbinZhou-NOAA ***@***.***>
> wrote:
>
> > Alicia,
> >
> > Done. Please double check it.
> >
> > Binbin
> >
> > On Thu, Jan 5, 2023 at 2:03 PM Alicia Bentley ***@***.***>
> > wrote:
> >
> > > Hi Binbin. I checked the contents of your .htaccess document, and it
> only
> > > contains "+ Indexes". Can you please copy/paste the text below into
> your
> > > .htaccess document instead? This will allow me to view a list of your
> > final
> > > EVS images. Thanks!
> > >
> > > The text to copy/paste into .htaccess is:
> > > Options +Indexes
> > >
> > > ___________________________________
> > > *Alicia M. Bentley, Ph.D.*
> > > Physical Scientist | Model Evaluation Group
> > > <https://www.emc.ncep.noaa.gov/users/meg/home/>
> > > Global Model Verification Team Lead
> > > NOAA/NWS/NCEP/EMC (VPPPG Branch)
> > > Work Phone: ‪(301) 683-3783‬
> > >
> > >
> > > On Tue, Jan 3, 2023 at 3:53 PM BinbinZhou-NOAA ***@***.***>
> > > wrote:
> > >
> > > > Perry,
> > > >
> > > > 1) updated GHRSST data in dcom to the new data path
> > > > 2) added more domains in 24hr APCP verification conf files since they
> > > > are missing
> > > >
> > > > Binbin
> > > >
> > > >
> > > >
> > > >
> > > > On Tue, Jan 3, 2023 at 3:30 PM PerryShafran-NOAA ***@***.***>
> > > > wrote:
> > > >
> > > > > Binbin - can you describe the changes for this particular PR?
> > > > >
> > > > > Perry
> > > > >
> > > > > —
> > > > > Reply to this email directly, view it on GitHub
> > > > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370196597>,
> > or
> > > > > unsubscribe
> > > > > <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFJVNNZXBA3HEYLJC3TWQSD6XANCNFSM6AAAAAATP65RZI
> > > > >
> > > > > .
> > > > > You are receiving this because you were mentioned.Message ID:
> > > > > ***@***.***>
> > > > >
> > > >
> > > >
> > > > --
> > > >
> > > > Binbin Zhou
> > > >
> > > > Lynker at NOAA/NWS/NCEP/EMC
> > > >
> > > > 5830 University Research Ct.
> > > >
> > > > College Park, MD 20740
> > > >
> > > > ***@***.***
> > > >
> > > > 301-683-3683
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1370213859>,
> or
> > > > unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/APUWOBBYG7VX5DRGZI3CS6LWQSGVHANCNFSM6AAAAAATP65RZI
> > > >
> > > > .
> > > > You are receiving this because you commented.Message ID:
> > > > ***@***.***>
> > > >
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1372615528>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFNXCBVNSFNX7TSLL7LWQ4LHNANCNFSM6AAAAAATP65RZI
> > >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> >
> > Binbin Zhou
> >
> > Lynker at NOAA/NWS/NCEP/EMC
> >
> > 5830 University Research Ct.
> >
> > College Park, MD 20740
> >
> > ***@***.***
> >
> > 301-683-3683
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1372799762>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBDO5PJ6GWDERCLVSQTWQ44LNANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1372804896>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFO7YRBUVEYQTW4PMPDWQ4465ANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

PerryShafran-NOAA commented:
There's been a fair bit of back and forth here.  Is this code ready for testing?  

Thanks!

Perry

AliciaBentley-NOAA commented:
Hi Binbin,

Thanks! I've begun looking over your final images, and I wanted to send you
my first comment now before I get to everything else:

General Comments:

   1.

   Image names - When I was looking at your image directories, I noticed
   that “glb” (meaning global) appears in every image name, even when the
   image is of conus, nhem, shem, etc.. For example:

   evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.
   conus_glb.png

   <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.conus_glb.png>
   evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.
   g003_glb.png
   <https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.g003_glb.png>

evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.
nhem_glb.png
<https://www.emc.ncep.noaa.gov/bzhou/evs_plots/gens/grid2grid/evs.global_ens.acc.hgt.p500.last31days.fhrmean_valid00z_12z.init00z_12z.nhem_glb.png>

We would like the grid used and the region shown to be in the bolded
section of the image name. Please change the bolded text at the end of each
image name to be g003_conus, g003_glb, g003_nhem, etc.

Please make these changes to your graphics PR when you get the chance. I'll
be reviewing the images themselves and providing additional comments to you
later today. Thanks!

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
wrote:

> There's been a fair bit of back and forth here. Is this code ready for
> testing?
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBFL233LHZHQU7CN5LDWRV74HANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

BinbinZhou-NOAA commented:
Alicia,

  Sure, I'll make these changes.  These plots are just testing and primary,
not ready for PR yet.
There are still some issue for global ens graphics, particularly for NAEFS:
#6 Temperature (G2O): it should be G2G
#8 Dew point: since Canadian ensemble has no dew point, only GEFS has dew
point. CMC ens  and NAEFS
   should be removed
#9, #10, ACC/Abs Err for U/V at both 850 and 250mb levels only verified in
G2G. So changed to G2O?
   see GEFS table

For GEFS, the spatial plot is still not ready. Currently I am using Ben
Black's py script. But the quality of
 spatial map is not so good due to low resolution. Ben's script is better
for hi-resolution data. So I'll test
Mallory's py scripts

I think after both GEFS and NAEFS graphics are done, I'll make PR for them
together.

Binbin





On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
wrote:

> There's been a fair bit of back and forth here. Is this code ready for
> testing?
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJJFSDJZJ4ZCGKTYCLWRV74HANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Hi Binbin,

Thanks for that information! This PR includes many plotting scripts, so I
thought that you considered those plotting scripts to be in final form. If
you'd like Perry to check the prep/stats scripts only, can you either 1)
modify this PR to only include prep/stats or 2) submit another PR with just
the prep/stats? I don't want the unfinished graphics scripts to hold up the
whole process.

Also, for the spatial maps, please use the code that Mallory is using for
her spatial maps. We'd like all of the QPF spatial maps, for example, to
look the same so that they can be displayed on the same QPF vs. QPE webpage
and they will all look consistent.

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Tue, Jan 10, 2023 at 11:28 AM BinbinZhou-NOAA ***@***.***>
wrote:

> Alicia,
>
> Sure, I'll make these changes. These plots are just testing and primary,
> not ready for PR yet.
> There are still some issue for global ens graphics, particularly for NAEFS:
> #6 Temperature (G2O): it should be G2G
> #8 Dew point: since Canadian ensemble has no dew point, only GEFS has dew
> point. CMC ens and NAEFS
> should be removed
> #9, #10, ACC/Abs Err for U/V at both 850 and 250mb levels only verified in
> G2G. So changed to G2O?
> see GEFS table
>
> For GEFS, the spatial plot is still not ready. Currently I am using Ben
> Black's py script. But the quality of
> spatial map is not so good due to low resolution. Ben's script is better
> for hi-resolution data. So I'll test
> Mallory's py scripts
>
> I think after both GEFS and NAEFS graphics are done, I'll make PR for them
> together.
>
> Binbin
>
>
>
>
>
> On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
> wrote:
>
> > There's been a fair bit of back and forth here. Is this code ready for
> > testing?
> >
> > Thanks!
> >
> > Perry
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFJJFSDJZJ4ZCGKTYCLWRV74HANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377527728>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBBFOAJQ5TB7DFUTEADWRWEZZANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

BinbinZhou-NOAA commented:
OK,

  Thanks!

Binbin

On Tue, Jan 10, 2023 at 12:27 PM Alicia Bentley ***@***.***>
wrote:

> Hi Binbin,
>
> Thanks for that information! This PR includes many plotting scripts, so I
> thought that you considered those plotting scripts to be in final form. If
> you'd like Perry to check the prep/stats scripts only, can you either 1)
> modify this PR to only include prep/stats or 2) submit another PR with just
> the prep/stats? I don't want the unfinished graphics scripts to hold up the
> whole process.
>
> Also, for the spatial maps, please use the code that Mallory is using for
> her spatial maps. We'd like all of the QPF spatial maps, for example, to
> look the same so that they can be displayed on the same QPF vs. QPE webpage
> and they will all look consistent.
>
> Take care,
> Alicia
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Physical Scientist | Model Evaluation Group
> <https://www.emc.ncep.noaa.gov/users/meg/home/>
> Global Model Verification Team Lead
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Tue, Jan 10, 2023 at 11:28 AM BinbinZhou-NOAA ***@***.***>
> wrote:
>
> > Alicia,
> >
> > Sure, I'll make these changes. These plots are just testing and primary,
> > not ready for PR yet.
> > There are still some issue for global ens graphics, particularly for
> NAEFS:
> > #6 Temperature (G2O): it should be G2G
> > #8 Dew point: since Canadian ensemble has no dew point, only GEFS has dew
> > point. CMC ens and NAEFS
> > should be removed
> > #9, #10, ACC/Abs Err for U/V at both 850 and 250mb levels only verified
> in
> > G2G. So changed to G2O?
> > see GEFS table
> >
> > For GEFS, the spatial plot is still not ready. Currently I am using Ben
> > Black's py script. But the quality of
> > spatial map is not so good due to low resolution. Ben's script is better
> > for hi-resolution data. So I'll test
> > Mallory's py scripts
> >
> > I think after both GEFS and NAEFS graphics are done, I'll make PR for
> them
> > together.
> >
> > Binbin
> >
> >
> >
> >
> >
> > On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
> > wrote:
> >
> > > There's been a fair bit of back and forth here. Is this code ready for
> > > testing?
> > >
> > > Thanks!
> > >
> > > Perry
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFJJFSDJZJ4ZCGKTYCLWRV74HANCNFSM6AAAAAATP65RZI
> > >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> >
> > Binbin Zhou
> >
> > Lynker at NOAA/NWS/NCEP/EMC
> >
> > 5830 University Research Ct.
> >
> > College Park, MD 20740
> >
> > ***@***.***
> >
> > 301-683-3683
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377527728>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBBFOAJQ5TB7DFUTEADWRWEZZANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377606133>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJW5PVY4X2Q5EG6GB3WRWLWXANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Perry,

  The last PR for global_ens  is just for prep/stats, not including plots.
So you can test it.

Thanks!

Binbin

On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
wrote:

> There's been a fair bit of back and forth here. Is this code ready for
> testing?
>
> Thanks!
>
> Perry
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFJJFSDJZJ4ZCGKTYCLWRV74HANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

AliciaBentley-NOAA commented:
Hi Binbin,

I made most of the changes that you suggested in the NAEFS graphics
spreadsheet. I want to clarify one thing about the NAEFS graphics, though.
You previously said:

#9, #10, ACC/Abs Err for U/V at both 850 and 250mb levels only verified in
G2G. So changed to G2O? *see GEFS table*

When you are looking at the NAEFS graphics tab
<https://docs.google.com/spreadsheets/d/1rumS6wwSD1DDFKF8of9z57YQ5dh4rlX44f25p6d02Wg/edit?pli=1#gid=434716072>,
you should be referring back to the NAEFS stats tab
<https://docs.google.com/spreadsheets/d/1axdiBPHku2Ul10jSDYFQ2enx2tZf5tLx-3dEgAWAAEs/edit?pli=1#gid=806957557>,
not the GEFS stats tab. The NAEFS stats tab
<https://docs.google.com/spreadsheets/d/1axdiBPHku2Ul10jSDYFQ2enx2tZf5tLx-3dEgAWAAEs/edit?pli=1#gid=806957557>
asks for ACC/Abs Err for U/V at both 850 hPa and 250 hPa compared to RAOBs
(see rows 7 and 8). Please make sure that these G2O stats for GEFS, CMC
Ens, and NAEFS are being generated in your PR so that you can plot them
later. Thank you!

Take care,
Alicia
___________________________________
*Alicia M. Bentley, Ph.D.*
Physical Scientist | Model Evaluation Group
<https://www.emc.ncep.noaa.gov/users/meg/home/>
Global Model Verification Team Lead
NOAA/NWS/NCEP/EMC (VPPPG Branch)
Work Phone: ‪(301) 683-3783‬


On Tue, Jan 10, 2023 at 1:01 PM BinbinZhou-NOAA ***@***.***>
wrote:

> Perry,
>
> The last PR for global_ens is just for prep/stats, not including plots.
> So you can test it.
>
> Thanks!
>
> Binbin
>
> On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
> wrote:
>
> > There's been a fair bit of back and forth here. Is this code ready for
> > testing?
> >
> > Thanks!
> >
> > Perry
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AQBMPFJJFSDJZJ4ZCGKTYCLWRV74HANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> Binbin Zhou
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377646639>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APUWOBC2LD7DU5YZXVEXKZTWRWPWFANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

BinbinZhou-NOAA commented:
Alicia,

  The ACC/Abs Err in #7 is an issue. They can be done only after MET_11.0
is available.
So both ACC and MAE in #7 in NAEFS stats table should be changed to "hold"
before
MET_11 is installed on WCOSS2
It is my fault  not to inform you of that issue. But if is G2G like GEFS,
both ACC and MAE
can be done right now.

Binbin

Binbin



On Tue, Jan 10, 2023 at 1:14 PM Alicia Bentley ***@***.***>
wrote:

> Hi Binbin,
>
> I made most of the changes that you suggested in the NAEFS graphics
> spreadsheet. I want to clarify one thing about the NAEFS graphics, though.
> You previously said:
>
> #9, #10, ACC/Abs Err for U/V at both 850 and 250mb levels only verified in
> G2G. So changed to G2O? *see GEFS table*
>
> When you are looking at the NAEFS graphics tab
> <
> https://docs.google.com/spreadsheets/d/1rumS6wwSD1DDFKF8of9z57YQ5dh4rlX44f25p6d02Wg/edit?pli=1#gid=434716072
> >,
> you should be referring back to the NAEFS stats tab
> <
> https://docs.google.com/spreadsheets/d/1axdiBPHku2Ul10jSDYFQ2enx2tZf5tLx-3dEgAWAAEs/edit?pli=1#gid=806957557
> >,
> not the GEFS stats tab. The NAEFS stats tab
> <
> https://docs.google.com/spreadsheets/d/1axdiBPHku2Ul10jSDYFQ2enx2tZf5tLx-3dEgAWAAEs/edit?pli=1#gid=806957557
> >
> asks for ACC/Abs Err for U/V at both 850 hPa and 250 hPa compared to RAOBs
> (see rows 7 and 8). Please make sure that these G2O stats for GEFS, CMC
> Ens, and NAEFS are being generated in your PR so that you can plot them
> later. Thank you!
>
> Take care,
> Alicia
> ___________________________________
> *Alicia M. Bentley, Ph.D.*
> Physical Scientist | Model Evaluation Group
> <https://www.emc.ncep.noaa.gov/users/meg/home/>
> Global Model Verification Team Lead
> NOAA/NWS/NCEP/EMC (VPPPG Branch)
> Work Phone: ‪(301) 683-3783‬
>
>
> On Tue, Jan 10, 2023 at 1:01 PM BinbinZhou-NOAA ***@***.***>
> wrote:
>
> > Perry,
> >
> > The last PR for global_ens is just for prep/stats, not including plots.
> > So you can test it.
> >
> > Thanks!
> >
> > Binbin
> >
> > On Tue, Jan 10, 2023 at 10:46 AM PerryShafran-NOAA ***@***.***>
> > wrote:
> >
> > > There's been a fair bit of back and forth here. Is this code ready for
> > > testing?
> > >
> > > Thanks!
> > >
> > > Perry
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377468872>, or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AQBMPFJJFSDJZJ4ZCGKTYCLWRV74HANCNFSM6AAAAAATP65RZI
> > >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> >
> > Binbin Zhou
> >
> > Lynker at NOAA/NWS/NCEP/EMC
> >
> > 5830 University Research Ct.
> >
> > College Park, MD 20740
> >
> > ***@***.***
> >
> > 301-683-3683
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377646639>, or
> > unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/APUWOBC2LD7DU5YZXVEXKZTWRWPWFANCNFSM6AAAAAATP65RZI
> >
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/49#issuecomment-1377661465>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFK3XGQ3TYVNWRQIJ5LWRWRHZANCNFSM6AAAAAATP65RZI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
